gateway/api/migrations/0001_initial.py

```diff
@@ -94,14 +94,14 @@ class Migration(migrations.Migration):
                     "status",
                     models.CharField(
                         choices=[
-                            ("PENDING", "Pending"),
+                            ("INITIALIZING", "Initializing"),
```

Comment:
Migrations should not be changed. They are append only.
Since they are changing structure of schema of database, they can only modify state of db that was there before. 

You need to create new migration that will change schema from this types of statuses to new ones.

Reply:
I think handling the migration on client side, like you suggested, is a nice solution here. I don't even think it's really that hacky. We leave everything on the backend in Ray language for clarity, and we just convert to our terminology on client side. Makes a lot of sense to me.

I can also generate a new migration file if you like that solution better.